### PR TITLE
Updated the Skeleton

### PR DESCRIPTION
**Pull request by alexshiao99**

### CODE REVIEW

File: client/src/Components/Reviews/index.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React fom 'react';
```
Comment (alexshiao99) [inline client/src/Components/Reviews/index.jsx:2]: Obvious typo for 'fom'

**Review by ceads1618 — CHANGES_REQUESTED** [review]
This is a useless chunk of code that doesn't run

File: client/src/Components/Reviews/index.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (AdarshrajRav) [inline client/src/Components/Reviews/index.jsx]: hi

File: client/src/Components/Reviews/index.jsx (hunk 1 above)
Comment (AdarshrajRav) [inline client/src/Components/Reviews/index.jsx:2]: hi

**Review by AdarshrajRav — COMMENTED** [review]
check

File: client/src/Components/App.jsx [hunk 3]
```diff
@@ -6,14 +6,14 @@ import Reviews from './Reviews/index.jsx';
 
 const App = ({}) => {
 
-
+hereis useless code that wont run
 
   return (
   <div>
-    <Overview />
```
Comment (ceads1618) [inline client/src/Components/App.jsx]: Please review lines 13 through 17 for appropriate indentations.

File: client/src/Components/App.jsx [hunk 4]
```diff
@@ -6,14 +6,14 @@ import Reviews from './Reviews/index.jsx';
...
+          <Overview />
     <Related />
-    <Reviews />
-    <QA />
+      <Reviews />
+    <QA />;
```
Comment (ceads1618) [inline client/src/Components/App.jsx]: This is just a single comment

File: client/src/Components/App.jsx [hunk 5]
```diff
@@ -6,14 +6,14 @@ import Reviews from './Reviews/index.jsx';
 
 const App = ({}) => {
 
-
+hereis useless code that wont run
```
Comment (ceads1618) [inline client/src/Components/App.jsx]: This is another test

Comment (ceads1618) [inline client/src/Components/App.jsx]: Lastly checking to see how this format appears

**Review by ceads1618 — CHANGES_REQUESTED** [review]
Please review the three comments I made regarding your code.

**Review by ceads1618 — DISMISSED** [review]
Looks good to go!

**Review by AdarshrajRav — APPROVED** [review]
Looks great!

**Review by euniceyunjkim — COMMENTED** [review]
Update: great job correcting the errors! 

Please review the typo - should be "from".

**Pull request merged**